std/json.d

```diff
@@ -1725,8 +1727,11 @@ pure nothrow @safe unittest // issue 15884
     assert(test(30738.22));
 
     assert(test(1 + double.epsilon));
-    assert(test(-double.max));
     assert(test(double.min_normal));
+    static if (realInDoublePrecision)
+        assert(test(-double.max / 2));
+    else
+        assert(test(-double.max));
```

Comment:
Answering my own question (from [Relax std.json unittest some more](https://github.com/ldc-developers/phobos/commit/0d14f4ed441b4e70997e930d23ab47f26f7483d7#diff-7441f65d4929cc6c45a3094b58973fcdR1704)): "`test(-double.max)` results in an overflow and thrown exception in `std.conv.parse!(double, string)`."

I guess that's not too bad; at least it doesn't just silently corrupt data.